src/Renci.SshNet/Sftp/SftpFileStream.cs

```diff
@@ -166,6 +169,28 @@ public virtual byte[] Handle
         /// </value>
```

Comment:
This property lacks documentation.

Reply:
No change here?

Reply:
Oh, I guess it it inherits the documentation from the `Stream` class.